x/stake/types/validator.go

```diff
@@ -36,68 +36,69 @@ type Validator struct {
 	UnbondingHeight  int64     `json:"unbonding_height"` // if unbonding, height at which this validator has begun unbonding
 	UnbondingMinTime time.Time `json:"unbonding_time"`   // if unbonding, min time for the validator to complete unbonding
 
-	Commission            sdk.Dec `json:"commission"`              // XXX the commission rate of fees charged to any delegators
-	CommissionMax         sdk.Dec `json:"commission_max"`          // XXX maximum commission rate which this validator can ever charge
-	CommissionChangeRate  sdk.Dec `json:"commission_change_rate"`  // XXX maximum daily increase of the validator commission
-	CommissionChangeToday sdk.Dec `json:"commission_change_today"` // XXX commission rate change today, reset each day (UTC time)
+	Commission              sdk.Dec   `json:"commission"`                 // the commission rate charged to delegators
+	CommissionMax           sdk.Dec   `json:"commission_max"`             // maximum commission rate which validator can ever charge
+	CommissionMaxChangeRate sdk.Dec   `json:"commission_max_change_rate"` // maximum daily increase of the validator commission
+	CommissionChangeToday   sdk.Dec   `json:"commission_change_today"`    // total commission rate change today (reset each day)
```

Comment:
I don't think this variable is necessary. You can just limit a validator to one change every 24 hours, and when they do change validate its within the max change rate. You don't need to store the last delta change.

Reply:
I was informed by @rigelrozanski that we would allow multiple changes a day as long as those changes are within the constraints of `CommissionMaxChangeRate`. Also, `CommissionChangeToday` is not new.

Are you suggesting we change and simplify the spec?

Reply:
I think I'll go with this approach for now that you're suggesting @ValarDragon -- I'll amend if @rigelrozanski suggests otherwise.

Reply:
Awesome!

I have the PoV atm that lets table implementing convenience changes like that to postlaunch. (It is a good idea, but unnecessary atm)

Reply:
Great suggestion - One change per day sounds fine if it will simplify the codebase :)